command_helper/get_documentation.py

```diff
@@ -0,0 +1,86 @@
+from os import path
+try:

```

Comment:
That is done because the command_helper/* unit tests are run from Python 2 because I wanted to make sure that Selenium2Library keywords works. And S2L does not yet support for Python 3. But Sublime uses that helper from it's own Python 3 and therefore that `try:` is there.